app/controllers/waste_exemptions_engine/last_email_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class LastEmailController < ApplicationController
+    def show
+      if ENV["ENABLE_LAST_EMAIL_CACHE"] == "true"
```

Comment:
We have a page in waste carriers which we only use for testing purposes, but we have the logic to enable/disable it in the route instead: https://github.com/DEFRA/waste-carriers-renewals/blob/master/config/routes.rb#L5 That feels a bit simpler to me but I suppose you could argue that it clutters up the routes file. What do you think?

Reply:
Think I prefer the routes version, and that answers my question regards `handle_disabled` 😁

Reply:
Fixed in 15b3b13.